moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp

```diff
@@ -57,6 +58,8 @@ class RobotInterfacePython : protected py_bindings_tools::ROScppInitializer
     py_bindings_tools::ROScppInitializer()
   {
     robot_model_ = planning_interface::getSharedRobotModel(robot_description);
+    if (!robot_model_)
+      throw std::runtime_error("RobotInterfacePython: invalid robot model");
```

Comment:
[`getSharedRobotModel` only returns a pointer to a cached robot model.](https://github.com/ros-planning/moveit/blob/4ac0c7432d335f57aab6836cbcaaac3fccf4b6f9/moveit_ros/planning_interface/common_planning_interface_objects/src/common_objects.cpp#L90-L104)
There is another PR (#46) pending to avoid invalid models in first place. Indeed, we might want to add some error message there.